src/canvasPanelComponents/slideshow/src/components/CanvasNavigation/CanvasNavigation.js

```diff
@@ -1,9 +1,34 @@
-import React, { Component } from 'react';
+import React, { Component, useEffect } from 'react';
 import { withBemClass } from '@canvas-panel/core';
+import withLocation from '../withLocation/withLocation';
 
 import './CanvasNavigation.scss';
 
 class CanvasNavigation extends Component {
+  componentDidMount() {
+    if (this.props.addressable && this.props.hash) {
+      let slideId =
+        this.props.hash.includes('slide') && this.props.hash.split('=')[1];
```

Comment:
We could re-use `queryString.parse(this.props.hash)` like the `withLocation` component. It should return an object `{ slide: 1234 }`.